server.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const helmet = require("helmet");
+const morgan = require("morgan");
+
+const projectsRouter = require("./routers/projectsRouter.js");
+const actionsRouter = require("./routers/actionsRouter.js");
+
+const server = express();
+
+server.use(express.json());
+server.use(helmet());
+server.use(morgan("dev"));
+
+server.use("/api/projects", projectsRouter);
+server.use("/api/actions", actionsRouter);
+
```

Comment:
Good server.js file, I would only add a sanity check endpoint here
```suggestion
server.get("/", (req, res) => {
res.send("Sanity check");
});
```